leanblueprint/Packages/renderer_templates/blueprint.jinja2s

```diff
@@ -47,7 +47,7 @@ name: thmenv
     {% call modal('Lean declarations') %}
         <ul class="uses">
           {% for lean, url in obj.userdata.lean_urls %}
-          <li><a href="{{ url }}">{{ lean }}</a></li>
+          <li><a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ lean }}</a></li>
```

Comment:
I don't get why you want to impose the `target="_blank"` behavior. Why not assuming people know how they want to open links?

Reply:
Personally, I would always need the doc & source for a particular definition in a new tab, avoiding the disruption of navigating the dep graph or reading the blueprint. Choosing this every time is very inconvenient, especially on Mac without a mouse. I didn't think much on giving the choice to the readers, but provided an opinionated default.

While I'll need this default myself, I'm happy to revert these lines for the PR.

Reply:
Yes, I think this is too opinionated. And helping people to get away from Macs is always a good action. On a normal computer you simply middle-click on a link if you want to open it in a new tab. This is exactly the same effort as a left-click.

Reply:
In retrospect, indeed, it's too opinionated.

Instead, I'll add a CSS class `leandecl` to all Lean links, so one can easily add customized behaviors/styles to Lean links in JavaScript of their own blueprint repo. Is that more neutral and acceptable?

Reply:
Sure.

Reply:
Done in utensil/leanblueprint#2 .